plugins/experimental/statichit/statichit.cc

```diff
@@ -563,6 +563,12 @@ TSRemapInit(TSRemapInterface * /* api_info */, char * /* errbuf */, int /* errbu
 TSRemapStatus
 TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
 {
+  const TSHttpStatus txnstat = TSHttpTxnStatusGet(rh);
+  if (0 != static_cast<int>(txnstat) && txnstat != TS_HTTP_STATUS_OK) {
```

Comment:
Why not:
```
if (TS_HTTP_STATUS_NONE != txnstat && txnstat != TS_HTTP_STATUS_OK) {
```
?

Reply:
Good point. That's what I get for copy/pasting.

Reply:
Fixed. And added a bonus cleanup based on this feedback.